src/Lachain.Core/RPC/HTTP/Web3/TransactionServiceWeb3.cs

```diff
@@ -526,7 +529,7 @@ private string SignTransaction(JObject opts)
             var signedTx = MakeAndSignTransaction(opts);
             var transaction = signedTx.Transaction;
             var sign = signedTx.Signature;
-            var rawTx = transaction.RlpWithSignature(sign);
+            var rawTx = transaction.RlpWithSignature(sign, HardforkHeights.IsHardfork_8Active(_stateManager.LastApprovedSnapshot.Blocks.GetTotalBlockHeight()));
```

Comment:
If this tx is later added to the pool, then we should use GetTotalBlockHeight() + 1